app/email-incoming/index.ts

```diff
@@ -0,0 +1,172 @@
+  }
+
+  const data = await client.send(new ListUsersInGroupCommand(listUsersParams))
+
+  const userTypeData = data.Users?.find(
+    (user) => extractAttributeRequired(user, 'email') == userEmail
+  )
+  if (!userTypeData) {
+    await sendEmail(
+      userEmail,
+      'GCN Circular Submission Warning: Missing permissions',
+      'You do not have the required permissions to submit GCN Circulars. If you believe this to be a mistake, please fill out the form at https://heasarc.gsfc.nasa.gov/cgi-bin/Feedback?selected=kafkagcn, and we will look into resolving it as soon as possible.'
+    )
+    return
+  }
+
+  const userData = {
+    sub: extractAttributeRequired(userTypeData, 'sub'),
+    email: extractAttributeRequired(userTypeData, 'email'),
+    name: extractAttribute(userTypeData, 'name'),
+    affiliation: extractAttribute(userTypeData, 'custom:affiliation'),
+  }
+
+  const circular = {
+    dummy: 0,
+    createdOn: Date.now(),
+    subject: parsed.subject,
+    body: parsed.text,
+    sub: userData.sub,
+    submitter: formatAuthor(userData),
+  }
+
+  const newCircularId = await autoIncrement.put(circular)
+
+  // Send a success email
+  await sendEmail(
+    userEmail,
+    `Successfully submitted Circular: ${newCircularId}`,
+    `Your circular has been successfully submitted. You may view it at http://${domain}gcn.nasa.gov/circulars/${newCircularId}`
+  )
+}
+
+export async function handler(event: SNSEvent) {
+  // Get tables for autoincrement
+  const db = await tables()
+  const doc = db._doc as unknown as DynamoDBDocument
+  const tableName = db.name('circulars')
+  const counterTableName = db.name('auto_increment_metadata')
+  if (!tableName || !counterTableName) {
+    throw new Error('Could not find tables')
+  }
+  const dangerously =
+    (await (db._db as unknown as DynamoDB).config.endpoint?.())?.hostname ==
+    'localhost'
+
+  const autoIncrement = new DynamoDBAutoIncrement({
+    doc,
+    counterTableName,
+    counterTableKey: { tableName: 'circulars' },
+    counterTableAttributeName: 'circularId',
+    tableName: tableName,
+    tableAttributeName: 'circularId',
+    initialValue: 1,
+    dangerously,
+  })
+
+  const handlerPromises = event.Records.map((record) =>
+    handleRecord(record, autoIncrement)
+  )
+  const results = await Promise.allSettled(handlerPromises)
+  const rejections = results.filter(isRejected).map(({ reason }) => reason)
+  if (rejections.length) throw rejections
+}
+
+async function sendEmail(recipient: string, subject: string, body: string) {
+  const sendEmailCommand = new SendEmailCommand({
+    Destination: {
+      ToAddresses: [recipient],
+    },
+    FromEmailAddress: `no-reply@${domain}gcn.nasa.gov`,
+    Content: {
+      Simple: {
+        Subject: {
+          Data: subject,
+        },
+        Body: {
+          Text: {
+            Data: body,
+          },
+        },
+      },
+    },
+  })
+
+  await sesClient.send(sendEmailCommand)
```

Comment:
```suggestion
  await sesClient.send(new SendEmailCommand({
    Destination: {
      ToAddresses: [recipient],
    },
    FromEmailAddress: `no-reply@${domain}gcn.nasa.gov`,
    Content: {
      Simple: {
        Subject: {
          Data: subject,
        },
        Body: {
          Text: {
            Data: body,
          },
        },
      },
    },
  })
)
```

and re-run prettier.